CMake/Utilities.cmake

```diff
@@ -36,30 +36,30 @@ function(build_dependency lib_name)
   # library building cmake.
   set(build_args ${ARGN})
 
-  file(REMOVE_RECURSE ${KINESIS_VIDEO_OPEN_SOURCE_SRC}/lib${lib_name})
+  file(REMOVE_RECURSE ${OPEN_SRC_INSTALL_PREFIX}/lib${lib_name})
 
   # build library
   configure_file(
-    ${CMAKE_SOURCE_DIR}/CMake/Dependencies/lib${lib_name}-CMakeLists.txt
-    ${KINESIS_VIDEO_OPEN_SOURCE_SRC}/lib${lib_name}/CMakeLists.txt COPYONLY)
+    ./CMake/Dependencies/lib${lib_name}-CMakeLists.txt
```

Comment:
Does cmake have a pre-configured variable for this path?

Reply:
AFAIK, this path is custom to us, so I don't think CMAKE have a pre-configure variable for this.